packages/kiota_serialization_form/lib/src/form_serialization_writer_factory.dart

```diff
@@ -0,0 +1,19 @@
+part of '../kiota_serialization_form.dart';
+
+/// Represents a serialization writer factory that can be used to create a form
+/// url encoded serialization writer.
+class FormSerializationWriterFactory implements SerializationWriterFactory {
+  @override
+  SerializationWriter getSerializationWriter(String contentType) {
+    if (contentType != validContentType) {
```

Comment:
Can I assume the comparison here is case insensitive?

Reply:
Uh no, should it be?

Reply:
It probably should https://github.com/microsoft/kiota-serialization-form-dotnet/blob/0496baaa5b4ec4a9e9749b82844973134d7bed36/src/FormParseNodeFactory.cs#L20

Reply:
Fixed